tests/lit-tests/3126.ispc

```diff
@@ -0,0 +1,15 @@
+// RUN: not %{ispc} --target=unknown --nowrap %s -o - 2>&1 | FileCheck %s --check-prefix=CHECK-TARGET
+// RUN: not %{ispc} --target-os=unknown --nowrap %s -o - 2>&1 | FileCheck %s --check-prefix=CHECK-OS
+// RUN: not %{ispc} --arch=unknown --nowrap %s -o - 2>&1 | FileCheck %s --check-prefix=CHECK-ARCH
+// REQUIRES: ARM_ENABLED && X86_ENABLED && LINUX_HOST && !WASM_ENABLED && !XE_ENABLED
```

Comment:
That's quite restrictive. Should we have more tests, which have run everywhere, but check less (i.e. not matching the whole string, but only portions of it)?

Reply:
I think checking the beginning of the string with targets is sufficient. I updated the test and added comments.

Reply:
Looks good!